### PR TITLE
[INLONG-2930][Manager] Add ClickHouse sink support in manager client

### LINKED ISSUE #2930: [Feature]Add clickhouse sink support in manager-client
### Description

Add clickhouse sink support in manager-client

### Use case

Add clickhouse sink support in manager-client

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gong**
### [INLONG-2930][manager]Add clickhouse sink support in manager client

Fixes #2930 

### Motivation

Add clickhouse sink support in manager client

### CODE REVIEW

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongStreamSinkTransfer.java [hunk 1]
```diff
@@ -50,6 +55,9 @@ public static SinkRequest createSinkRequest(StreamSink streamSink, InlongStreamI
         if (sinkType == SinkType.KAFKA) {
             return createKafkaRequest(streamSink, streamInfo);
         }
+        if (sinkType == SinkType.CLICKHOUSE) {
```
Comment (healchow) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongStreamSinkTransfer.java:58]: Please use `else if` statement.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/DefaultInlongStreamBuilder.java [hunk 2]
```diff
@@ -182,7 +183,9 @@ private int initOrUpdateSource(SourceRequest sourceRequest) {
 
     private int initOrUpdateSink(SinkRequest sinkRequest) {
         String sinkType = sinkRequest.getSinkType();
-        if (SinkType.HIVE.name().equals(sinkType) || SinkType.KAFKA.name().equals(sinkType)) {
+        boolean flag = SinkType.HIVE.name().equals(sinkType) || SinkType.KAFKA.name().equals(sinkType)
+                || SinkType.CLICKHOUSE.name().equals(sinkType);
+        if (flag) {
```
Comment (healchow) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/DefaultInlongStreamBuilder.java:188]: Great.

**Review by kipshi — APPROVED** [review]

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongStreamSinkTransfer.java (hunk 1 above)
Comment (gong) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/util/InlongStreamSinkTransfer.java:58]: ok

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/thirdparty/sort/util/SinkInfoUtils.java [hunk 3]
```diff
@@ -76,13 +80,53 @@ public static SinkInfo createSinkInfo(InlongStreamInfo inlongStreamInfo, SourceR
         String sinkType = sinkResponse.getSinkType();
         if (SinkType.forType(sinkType) == SinkType.HIVE) {
             return createHiveSinkInfo(sourceResponse, (HiveSinkResponse) sinkResponse);
-        } else if (SinkType.forType(sinkType) == SinkType.KAFKA) {
+        }
+        if (SinkType.forType(sinkType) == SinkType.KAFKA) {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/thirdparty/sort/util/SinkInfoUtils.java:84]: Please use `else if`, not all `if` statement.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/thirdparty/sort/util/SinkInfoUtils.java [hunk 4]
```diff
@@ -76,13 +80,53 @@ public static SinkInfo createSinkInfo(InlongStreamInfo inlongStreamInfo, SourceR
...
+        }
+        if (SinkType.forType(sinkType) == SinkType.CLICKHOUSE) {
+            return createClickhouseSinkInfo(inlongStreamInfo, sourceResponse, (ClickHouseSinkResponse) sinkResponse);
+        }
+        // todo iceberg is wait to support
+        throw new RuntimeException(
+                String.format("SinkType:{} not support in CreateSortConfigListener", sinkType));
+    }
+
+    private static ClickHouseSinkInfo createClickhouseSinkInfo(InlongStreamInfo inlongStreamInfo,
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/thirdparty/sort/util/SinkInfoUtils.java:95]: Are you sure you need a `private static` method?

Comment (gong) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/thirdparty/sort/util/SinkInfoUtils.java:95]: > Are you sure you need a `private static` method?

only for `createSinkInfo` to use

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/thirdparty/sort/util/SinkInfoUtils.java (hunk 3 above)
Comment (gong) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/thirdparty/sort/util/SinkInfoUtils.java]: done

**Review by healchow — APPROVED** [review]

**Pull request merged**